packages/mdc-toolbar/README.md

```diff
@@ -0,0 +1,111 @@
+# MDC Toolbar
+
+MDC Toolbar provides an `mdc-toolbar` class for easily styling a RTL-aware
+toolbar. The toolbar scrolls with content by default, but supports fixed on
+top as well.
+
+
+## Installation
+
+```
+npm install --save @material/toolbar
+```
+
+
+## Usage
+
+Wrap the items with `mdc-toolbar` class in following way:
+
+```html
+<div class="mdc-toolbar">
+  <div class="mdc-toolbar__section">...</div>
+</div>
+```
+
+### Fix to top
+
+By default, toolbar scroll with the content, you can add `mdc-toolbar_fixed` to
+fix toolbar to the top of screen.
+
+**Body margin required**
```

Comment:
Thanks for pointing this out. The original design was mainly about `mdc-toolbar-fixed-adjust` and `mdc-toolbar--fixed` could only be used together to prevent user forget adding proper `margin-top`, thus the nested structure. But I feel you are right, this imposes user much have such a DOM structure and less flexible and that mistake is trivial compare to this.